src/main/java/miro/main/Miro.java

```diff
@@ -0,0 +1,54 @@
+package miro.main;
+
+import java.util.Scanner;
+
+public class Miro {
+    private boolean isExit;
+    private final Scanner sc = new Scanner(System.in);
+    private TaskList taskList;
+    private final Ui ui;
+    private final Parser parser;
+    private String miroResponse;
+
+    public Miro() {
+        this.ui = new Ui();
+        this.isExit = false;
+        String filepath = "./data/duke.txt";
```

Comment:
would recommend to use `static final` to declare a constant

```suggestion
        private static final String FILE_PATH = "./data/duke.txt";
```

Reply:
Agree. Also since it's a constant name, it should be in SCREAMING_SNAKE_CASE.